swagger.yaml

```diff
@@ -304,59 +307,87 @@ definitions:
         description: "Description for a permission"
         type: string
         example: "read only"
+  Conditions:
+    type: object
```

Comment:
My understanding was that conditions would be an array. Each individual condition would contain a single attribute / value?

Reply:
Conditions should be array. But attributes and values can be array as well too as suggested by @janderson2 
Basically we are trying to borrow the AWS IAM policies: https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_multi-value-conditions.html